pkg/pb/inspector.proto

```diff
@@ -199,3 +207,26 @@ message DashboardResponse {
   google.protobuf.Timestamp last_pinged = 8;
   google.protobuf.Timestamp last_queried = 9;
 }
+
+message SegmentHealthRequest {
+  bytes encrypted_path = 1;
```

Comment:
add `bucket` and segment index here as well, similarly how metainfo api defines them

Reply:
If we are given the path why would we need them?

Reply:
done